homeassistant/components/sensor/tahoma.py

```diff
@@ -0,0 +1,62 @@
+"""
+Support for Tahoma sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.tahoma/
+"""
+
+import logging
+
+from datetime import timedelta
```

Comment:
Please group the imports (see https://www.python.org/dev/peps/pep-0008/#imports).